cypress/e2e/felony/utils/testIneligiblePage.ts

```diff
@@ -0,0 +1,19 @@
+export default function testIneligiblePage(link: string) {
+  describe("f-classa-main-1-ineligble", () => {
+    it('has ineligibility content', () => {
+      cy.visit(`/calculator/${link}`);
+      cy.contains('Your conviction is not eligible to vacate')
+        .should('be.visible')
+      cy.contains('as defined in RCW 9.94A.030')
```

Comment:
```suggestion
      cy.get('a:visible')
```
Same here, lets try to not be coupled on texts, because they can change easily, what do you think to use `a` element and just check if there is the right href?